### PR TITLE
Removed deprecated code from StorkApp.

### LINKED ISSUE #24279: stork.sh uses deprecated code
## Bug Description
scripts/stork.sh uses the deprecated function StorkApp::registerAll instead of the new ModulesApp::registerAllObjects<StorkApp>.

## Steps to Reproduce
Use stork.sh to create a new app and run a problem. Note that the deprecated function StorkApp::registerAll is used.

## Impact
Annoying.

### PR DESCRIPTION
**Pull request by pbehne**
Replaced the deprecated StorkApp::registerAll with ModuleApp::registerAllObjects<StorkApp>.

Closes #24279.

### CODE REVIEW

**Review by GiudGiud — DISMISSED** [review]

**Comment by pbehne** [comment]
@GiudGiud, I should be finished now. Just caught some typos.

**Review by GiudGiud — APPROVED** [review]

**Comment by GiudGiud** [comment]
Sorry for forgetting about that one. Please ping me if you ever see it happen again

**Comment by loganharbour** [comment]
giphy

File: stork/src/base/StorkApp.C.app [hunk 1]
```diff
@@ -14,21 +14,11 @@ StorkApp::validParams()
...
-void
-StorkApp::registerAll(Factory & f, ActionFactory & af, Syntax & syntax)
-{
-  ModulesApp::registerAll(f, af, syntax);
-  Registry::registerObjectsTo(f, {"StorkApp"});
-  Registry::registerActionsTo(af, {"StorkApp"});
-
-  /* register custom execute flags, action syntax, etc. here */
-}
-
```
Comment (cticenhour) [inline stork/src/base/StorkApp.C.app:31]: This was not the right fix. `ModulesApp::registerAll` should have been modified directly. Stork is now broken for new users.

Comment (cticenhour) [reply stork/src/base/StorkApp.C.app:31]: @Naktakala should have a PR up fixing this shortly.

**Comment by loganharbour** [comment]
In the future - please ping Cody for registration changes

**Comment by cticenhour** [comment]
Alright, I will make a note to fix it. Thanks for letting me know.

File: stork/src/base/StorkApp.C.app (hunk 1 above)
Comment (pbehne) [reply stork/src/base/StorkApp.C.app:31]: This change was made based on the existing deprecation message for `StorkApp::registerAll`.

**Pull request merged**